repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/CaseTypeLiteEntity.java

```diff
@@ -0,0 +1,117 @@
+package uk.gov.hmcts.ccd.definition.store.repository.entity;
+
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import static javax.persistence.CascadeType.ALL;
+import static javax.persistence.FetchType.EAGER;
+import static javax.persistence.GenerationType.IDENTITY;
+
+/**
+ * A "lite" version of the {@link CaseTypeEntity} class that contains selected Case Type fields (id, reference, name,
+ * description, and list of states) for display purposes. (Class introduced to avoid loading the whole CaseTypeEntity,
+ * which is too expensive to do for all Case Types of a given Jurisdiction.)
+ */
+@Table(name = "case_type")
+@Entity
+public class CaseTypeLiteEntity implements Serializable, Versionable {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = IDENTITY)
+    private Integer id;
+
+    @Column(name = "reference", nullable = false)
+    private String reference;
+
+    @Column(name = "version", nullable = false)
+    private Integer version;
+
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @Column(name = "description")
+    private String description;
+
+    @ManyToOne(fetch = EAGER)
```

Comment:
I believe the jurisdiction could be lazy as it's most likely not consumed outside of the JPQL query